site/en/r2/tutorials/distribute/training_loops.ipynb

```diff
@@ -702,7 +702,22 @@
         "id": "MuZGXiyC7ABR"
       },
       "source": [
-        "Note: We do not use `tf.metrics.Mean` to track the training loss across different replicas because of how we scale the training loss. For example, let us assume that we have a global batch size of 4 and two devices. Two samples are processed on each of the devices. Let the loss values be [2,  3] and [4,  5] on each of the two devices. With loss scaling, the value of loss on each replica is 5/4=1.25 and 9/4=2.25. Note that we divide by 4 which is the global batch size. If we use `tf.metrics.Mean` to track loss we end up with a `total` of 3.50 and `count` of 2 which results in  `total`/`count`=1.75  when `result()` is called on the metric. The `tf.keras.Metrics` loss is scaled by an additional factor which is equal to the number of replicas in sync. As a general rule you should use `tf.keras.Metrics` to track per sample values and avoid values that have been aggregated within a replica."
+        "### Tracking training loss across replicas\n",
+        "\n",
+        "As a general rule, you should use `tf.keras.Metrics` to track per-sample values and avoid values that have been aggregated within a replica. Loss scaling with `tf.keras.Metrics` accounts for the number of replicas in sync.\n",
+        "\n",
+        "We do *not* recommend using `tf.metrics.Mean` to track the training loss across different replicas, because it does not account for the number of replicas in\n",
```

Comment:
"because it does not account for the number of replicas" can be replaced with "because of the loss scaling computation that we carry out" or something to that effect.